public/app/plugins/datasource/dashboard/datasource.test.ts

```diff
@@ -87,20 +94,20 @@ describe('DashboardDatasource', () => {
     expect(sourceData.isActive).toBe(false);
   });
 
-  it('Should emit only the first value and complete if used within MixedDS', async () => {
+  it('Should emit only the first value and complete if used', async () => {
     const { observable } = setup({ refId: 'A', panelId: 1 }, `${MIXED_REQUEST_PREFIX}1`);
```

Comment:
I think `MIXED_REQUEST_PREFIX` should be removed here? Otherwise this test is the same as the next one.